CHANGELOG.md

```diff
@@ -7,6 +7,15 @@ Changelog is rather internal in nature. See release notes for the public overvie
 
 ## Version 2.0.0
 
+- [#478]
+  - **Description:** Changed _dev-only to dev-only
+  - **Products impact:** Bug fix
```

Comment:
@BabyElias  Could you please change this to `Script enhancement` and not `Bug fix`.

Reply:
Done.